tike/tomo.py

```diff
@@ -112,34 +112,29 @@
 logger = logging.getLogger(__name__)
 
 
-def _tomo_interface(obj, obj_corner,
-                    probe, theta, v, h,
-                    **kwargs):
+def _tomo_interface(
+    obj, obj_corner,
+    probe, theta, v, h,
```

Comment:
Is the order of theta, v, h are the same as the way data is written? data.shape ?= [len(theta), len(v), len(h)]

Reply:
It should be? We discussed in-person that `tomopy` has this (theta, v, h) convention due to the fact that people usually collect data as horizontal being the fast axis.

Reply:
Yes. If you read the documentation at the top of the page, H is the fastest axis.